client/src/components/graph/graph.tsx

```diff
@@ -922,6 +924,21 @@ class Graph extends React.Component<GraphProps, GraphState> {
       offsetY = (offscreenCanvas.height - targetHeight) / 2;
     }
 
+    /**
+     * (thuang): In test mode, we only want to show a bit of the image underlay
+     * to prevent Chromatic snapshots from creating false positives.
```

Comment:
do you wanna include cropping in the comment here?

Reply:
Great idea 🙏 

Updated to this:

```ts
/**
     * (thuang): In test mode, we crop the image underlay and the graph
     * to prevent Chromatic snapshots from creating false positives.
     */
```

Thank you!